docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,29 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 20th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using Fivetran and its [Stripe connector](https://fivetran.com/docs/applications/stripe). This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It also includes information regarding subscriptions and customers, as well as products, plans, invoices, cards, charges, discounts, coupons, and promotion codes.
+
+Some of the customer information includes the customer's postal code, state, and country for a subscription, which helps resolve issues for customers across a particular dimension (e.g., country, payment provider, etc.). However, as it intentionally excludes any personal identifiable information of customers (e.g., name, street address, phone number, email address), it can be challenging to use to help on an individual customer level. Other uses for this data is for retrieving all-time counts (e.g., total number of active subscriptions).
+
+Fivetran preserves historical data within `subscription_history` by comparing the current state of the subscription with the most recent record in the history table and adding a new record if they vary.
+
+### Mobile app store providers
+Subscriptions Services data is also obtained from two mobile app store providers: Apple and Google. This includes date of billing and expired dates, subscription change event, etc. Apple provides limited data - it does not provide the country, price, or currency of subscriptions.
+
+Subscription Platform handles getting Apple subscriptions from the Apple API and stores the latest state of active subscriptions in Firestore. Prior to December 2022, VPN Apple subscriptions data was managed by VPN's Guardian service and stored in the Guardian database. As Firestore doesn't contain the full historical state of Apple subscriptions, Guardian data is still used for historical purposes.
```

Comment:
Nit: s/Apple API/App Store Server API

I'm wondering if we can shorten the VPN note about using the Guardian DB in addition to SubPlat's Firestore DB. Ideally, we want statements here to be generally true/applicable to all RPs with minimal RP-specific asides. Granted, Sean's talk had a heavy emphasis on VPN-specific details, because they were the first major SubPlat product to go through the ringer here. I would imagine other product teams won't be in this situation of having older subscription data.

Maybe something like: As of December 2022, SubPlat supports [Apple IAP](link to our docs on Apple IAP] subscriptions, with current and historical subscription state from that point forward stored in our Firestore DB and synced in BigQuery.